app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,60 @@
+<div id="<%= dom_id recipe %>">
+  <p>
+    <%# <strong>Name:</strong> %>
+    <%= recipe.name %>
+  </p>
+
+  <p>
+    <%# <strong>Preparation time:</strong> %>
+    <%= recipe.preparation_time %>
+  </p>
+
+  <p>
+    <%# <strong>Cook time:</strong> %>
+    <%= recipe.cook_time %>
+  </p>
+
+  <p>
+    <strong>Steps go here...</strong>
+    <%= recipe.description %>
+  </p>
+  <% if recipe.user_id == current_user.id %>
+    <%= link_to "Generate shopping list", shopping_list_index_path %>
+  <% end %>
+  <%= link_to "Add ingredient", new_recipe_recipe_food_path(recipe) %>
+  <% if can? :update, recipe %>
+    <p>
+      <%= button_to(recipe.public ? "Make Private" : "Make Public", toggle_public_recipe_path(recipe), method: :put, remote: true) %>
+      <%# <strong>Public:</strong> %>
+      <%#= recipe.public %>
+    </p>
+  <% end %>
+
+  <table>
+    <tr>
+      <th>Food</th>
+      <th>Quantity</th>
+      <th>Value</th>
+      <th>Actions</th>
+    </tr>
+    <% @recipe_foods.each do |recipe_food| %>
+      <tr>
+        <td><%= recipe_food.food.name %></td>
+        <td><%= recipe_food.quantity %></td>
+        <td><%= recipe_food.food.price * recipe_food.quantity %></td>
+        <td>
+          <% if can? :destroy, recipe %>
+            <%= link_to "Modify", edit_recipe_recipe_food_path(recipe, recipe_food) %>
+            <%= button_to "Remove", recipe_recipe_food_path(recipe, recipe_food), method: :delete %>
+          <% end %>
+        </td>
+      </tr>
+    <% end %>
+  </table>
+
+  <%# <p> %>
+    <%# <strong>User:</strong> %>
+    <%#= recipe.user_id %>
+  <%# </p> %>
+
+</div>
```

Comment:
- [ ] Let's also ensure a proper layout for a Recipe detail.

**Your design:**        |  **Expected:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Chandan-devs-tech/recipe-app/assets/12745474/975cd284-3db1-40cc-a701-1b89d2f54624"> | <img width="800" alt="Home Page" src="https://github.com/cancelei/recipe-in-rails/assets/12745474/eeffee25-fc28-49e8-9dcf-151d291ad847">